### PR TITLE
Refine modal component

### LINKED ISSUE #642: UI - refine the modal component (cross-platform)
Screen Shot 2021-03-03 at 3 12 30 PM

- Not currently optimized for mobile
- Needs UI refresh

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-114)

### PR DESCRIPTION
**Pull request by flozia**
Refines the modal component that is displayed when a device is removed in the “My devices” view. 

**Design**
[Refine Modal UI](https://www.figma.com/file/Pv2ub3IqQt4CaDPwhFTyst/Refine-Modal-UI-Device-Management) Figma file

**Screenshots**
MacOS
remove_device_popup-macOS

Resolves #642

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-781)

### CODE REVIEW

File: src/ui/components/VPNPopupButton.qml [hunk 1]
```diff
@@ -35,11 +35,11 @@ VPNButtonBase {
     contentItem: VPNInterLabel {
```
Comment (lesleyjanenorton) [inline src/ui/components/VPNPopupButton.qml:35]: Could this also become VPNMetropolisLabel {} ?

**Review by lesleyjanenorton — COMMENTED** [review]
This is looking and working great - left just a few comments!

**Comment by flozia** [comment]
Updated the modal buttons according to the [recently updated design](https://github.com/mozilla-mobile/mozilla-vpn-client/issues/642#issuecomment-884620498).
modal-stacked-buttons

File: src/ui/components/VPNPopupButton.qml (hunk 1 above)
Comment (flozia) [reply src/ui/components/VPNPopupButton.qml:37]: According to the design change it seems like the _cancel_ and _remove_ buttons have a different font now and the `font.family` property is set dynamically now.

File: src/logo_beta.qrc [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <RCC>
     <qresource prefix="/">
         <file>ui/resources/logo-dock-beta.png</file>
+        <file>ui/components/VPNMetropolisLabel.qml</file>
```
Comment (lesleyjanenorton) [inline src/logo_beta.qrc:4]: We should remove VPNMetroplisLabel.qml here and add it `src/qml.qrc`

**Review by lesleyjanenorton — COMMENTED** [review]
Almost there! We just need to add `VPNMetroplisLabel.qml` to `src/qml.qrc` and fix the conflict in that file and then we are good to go here! Thank you!

**Review by lesleyjanenorton — COMMENTED** [review]
Looks great - we just need to pick out the i18n file

**Review by lesleyjanenorton — APPROVED** [review]
Thank you @flozia!

**Pull request merged**